language/documentation/book/src/generics.md

```diff
@@ -239,7 +239,7 @@ In the first one,
 the parameter `T1` is not used at all inside the struct definition.
 In the second one, the parameter `T1` is only used as an argument to a phantom type parameter.
 
-```move=
+```move
```

Comment:
Why the removal of the line numbers?

Reply:
Are you saying that the equals sign can be used to render line numbers in mdBook?